### ISSUE TITLE
Simply skill's passive attacking non hostile mob around

### ISSUE THREAD

**Issue by thearmygeddon** [Sweenus/SimplySkills#117]
Simply skill's passive attacking non hostile mob around spesificly that i've been using raging javelin, swordfall and divine adjudication whenever those passive trigger it attacks any nearby mob exclude pets

**Comment by megumew** [comment]
Yes please add some sort of configuration to this. If I activate it accidentally near my farm, the entire farm is wiped in seconds, other player's pets murdered. Even other friendly players killed. It makes this mod unable in groups currently and I am trying to brainstorm how I am going to fix this for my server.

 If you have any options like other mods to prevent this please let me know.
